app/modules/events/card.tsx

```diff
@@ -0,0 +1,17 @@
+import type { Event } from '@prisma/client';
+
+export function EventCard({ name, date, description, imgUrl, imgAlt, location }: Event) {
+  return (
+    <div className="grid grid-cols-3 gap-8 p-4 bg-primary text-white rounded-xl">
+      <div className="">
+        <img src={imgUrl ?? ''} alt={imgAlt ?? 'group image'}></img>
```

Comment:
We shouldn't even render the image if there is no `imgUrl` and `imgAlt`:

```suggestion
        {{imgUrl && imageAlt && <img src={imgUrl} alt={imgAlt}></img>}}
```